wordpress-mobile/WordPress-iOS
# ActivityLog: add detail view.

Adds detail view to display after tapping on an item in Activity Log. 

(after tearing my hairs out debugging autolayout/tableView issue here for 3h I realised that this isn't necessarily a tableView and after rewriting this into bunch of `UIStackView`s, it's now magic)

Here's how it looks like: 

![simulator screen shot - iphone 8 plus - 2018-04-20 at 09 40 47](https://user-images.githubusercontent.com/1594081/39036870-2ac0ee5a-447f-11e8-8e49-218618eef12c.png)
![simulator screen shot - iphone 8 plus - 2018-04-20 at 09 40 28](https://user-images.githubusercontent.com/1594081/39036871-2ad9a68e-447f-11e8-8aad-4ffa743c90a4.png)



To test:
1. Open activity log
2. Tap on an item
3. Verify that it looks like on designs
